SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/GridBasedLBStrategy/GridBasedLBStrategyTest.cpp

```diff
@@ -205,14 +230,16 @@ GRIDBASEDLBSTRATEGY_TEST(GIVEN_grid_is_not_ready_WHEN_local_virtual_worker_id_is
 
 	TestTrue("IsReady After LocalVirtualWorkerId Set", Strat->IsReady());
 
+	Cleanup();
```

Comment:
Is there a reason for calling this directly, and not using the existing LATENT cmd?

Reply:
There isn't a good reason, no; I thought it improper to add latent commands where there isn't an explicit need for them - namely, if there were no latent commands in a test, I'd opted not to add them.
As an anecdote, changing this line to `ADD_LATENT_AUTOMATION_COMMAND(FCleanup());` doubles this test's run time on my machine.